search/src/Components/upload/upload_stepper.js

```diff
@@ -352,6 +369,22 @@ class UploadStepper extends React.Component {
                 <Typography className={classes.instructions}>
                     {this.getStepContent(stepName)}
                 </Typography>
+                {stepName === "images" ?
```

Comment:
I'm having trouble returning array from a functional component. When I try to decouple them, React gives me:
```
Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.
```

Reply:
Is this in relation to the code you've currently pushed?

Reply:
I think I made it work now with the latest commit, but returning array of jsx elements from functional component will render them in a row, not allowing decoupling.

Reply:
That doesn't make sense to me... not sure how component structure should affect presentation. But I can't investigate now.